sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestCase.java

```diff
@@ -19,9 +19,9 @@
  */
 package org.sonar.api.test;
 
-import javax.annotation.Nullable;
-
 import java.util.List;
+import javax.annotation.Nullable;
+import org.sonar.api.batch.fs.InputFile;
 
 public interface MutableTestCase extends TestCase {
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png)"TestCase" is deprecated, implement the suggested replacement instead.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1724)